radio/src/lua/api_general.cpp

```diff
@@ -1460,16 +1476,32 @@ Play a numerical value (text to speech)
  * `0 or not present` plays integral part of the number (for a number 123 it plays 123)
  * `PREC1` plays a number with one decimal place (for a number 123 it plays 12.3)
  * `PREC2` plays a number with two decimal places (for a number 123 it plays 1.23)
+ 
+ @param volume (number):
+ - (-2..2) override radio setting Wav volumne for the duration of file
+ - omitting the parameter uses radio setting Wav volume
 
-@status current Introduced in 2.0.0
+@retval none 
+
+@status current Introduced in 2.0.0, changed in 2.10
 
+// targets: BW, COLOR
+//
+// EXAMPLES:
+// playNumber(123, 3, 0, 2)  -- play number 123, unit mAh, use Wav volume 2
+// playNumber(123, 3, 0, -1) -- play number 123, unit mAh, use Wav volume -1
+// playNumber(123, 3, 0)     -- play number 123, unit mAh, use Wav volume 0
```

Comment:
Isn't `playNumber(123, 3, 0)` using default volume?

Reply:
No. Consider these radio settings:
![image](https://github.com/EdgeTX/edgetx/assets/5615068/c7bc9b25-bed0-4454-b363-0f89db1fb359)
They are the "user default" settings (don't confuse this with the system default settings). 

playNumber(123,3,**0**) will effectively override the Wav volume setting for the playNumber duration to
![image](https://github.com/EdgeTX/edgetx/assets/5615068/ded843f4-5f02-40b8-b8cd-6329c9d65344)
which is different to the "user default" and different to the "system default" (as user changed overall volume too).

Reply:
But isn't volume the **4th** parameter? 
> playNumber(value, unit [, attributes [, volume]]

Reply:
@raphaelcoeffic @pfeerick I shouldn't respond in a hurry. Of course, omitting the last parameter doesn't override any of the user settings, ie the user radio settings Wav volume is applied. This also how compatibility is ensured.

Reply:
omitting the forth parameter is identical to setting the forth parameter to USE_SETTINGS_VOLUME. Omitting it is for backwards compatibility